travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
+
+
+def send_output(nick, dep, dest, date, result=None):
+    """Sends output"""
+    out = 'Nothing found'
+
+    if date == '':
+        date = datetime.datetime.today().strftime('%d.%m.%Y')
+
+    if len(result):
+        out = result[0].__repr__()
+        try:
+            time = result[0].lines[0].departure
+        except AttributeError:
+            time = result[0].drives[0].begin_time
+        searched[nick] = [dep, dest, time, date]
+
+    out = unicode(out.strip(codecs.BOM_UTF8), 'utf-8')
+    return out.encode('utf-8')
+
+
+def searched_incrementer(nick):
+    """Adds 1 minute to previous search departure"""
+    user = searched[nick]
```

Comment:
@Adman this really makes no sense. If you are using `user[0]`, `user[1]` and `user[2]` why don't you just do 

```python
dept, dest, time, date = searched[nick]
```

and use the respective variables? This is as unreadable as it gets.